services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualDataset/activatedLLMPromptsSection.tsx

```diff
@@ -168,6 +176,7 @@ const ActivatedLLMPromptsSection = ({ promptId, datasetId, individualDatasetLink
       guidelinesCount: guidelines_count,
       optimalAccuracy: percentAccuracy(optimal_correct, optimal_count),
       suboptimalAccuracy: percentAccuracy(suboptimal_correct, suboptimal_count),
+      feedbackErrorRate: evaluator_total_count === null ? null : percentAccuracy(evaluator_suboptimal_count, evaluator_total_count),
```

Comment:
It might be worth handling the null case within percentAccuracy so that you have standard behavior

Reply:
Yeah, that's a good idea, done!